include/grpcpp/security/tls_certificate_verifier.h

```diff
@@ -214,11 +214,32 @@ class ExternalCertificateVerifier {
       request_map_ ABSL_GUARDED_BY(mu_);
 };
 
+// A CertificateVerifier that will perform hostname verification, to see if the
+// target name set from the client side matches the identity information
+// specified on the server's certificate.
 class HostNameCertificateVerifier : public CertificateVerifier {
  public:
   HostNameCertificateVerifier();
 };
 
+// A sample ExternalCertificateVerifier(which is generally implemented by users)
+// that doesn't perform any additional checks other than certificate
+// verification, if specified.
+// Note: using this solely without any other authentication mechanisms on the
+// peer identity will leave your applications to the MITM(Man-In-The-Middle)
+// attacks. Users should avoid doing so in production environments.
+class NoOpCertificateVerifier : public ExternalCertificateVerifier {
```

Comment:
Why make this an external verifier?  If we're providing this out of the box, let's just implement it inside core, the same way that we do for the hostname verifier, so that we don't have an extra and unnecessary level of indirection.